common/src/python/projects/study_mapping.py

```diff
@@ -174,11 +226,23 @@ def __add_retrospective(self, center: CenterGroup) -> None:
         Args:
           center: the center group
         """
-        labels = [
-            f"retrospective-{datatype.lower()}" for datatype in self.__study.datatypes
-        ]
-        for label in labels:
-            center.add_project(label)
+        if not self.study.is_primary():
```

Comment:
Don't we need retrospective projects if the affiliated study has separately enrolled participants with UDSv3 data?

Reply:
Thanks for pointing this out. I am going to add an explicit "legacy" flag to indicate this instead of assuming only the primary study has legacy data.